src/anyp/Uri.cc

```diff
@@ -116,98 +117,80 @@ urlInitialize(void)
 }
 
 /**
- * Parse the scheme name from string b, into protocol type.
- * The string must be 0-terminated.
+ * Parse the URI scheme details from tokenizer.
```

Comment:
Done, though the input is not necessarily a URI. After the changes to AnyP::Uri::parse() in this PR it can be a raw I/O buffer from some parse() callers.